content/blog/2021-08-24-pre-defined-models-exp.md

```diff
@@ -0,0 +1,374 @@
+model. You should see results similar to this in your table.
+
+```dvctable
+┏━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━┳━━━━━━┳━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━━━━┳━━━━━━━┳━━━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━━━━┓
+┃ neutral:**Experiment**              ┃ neutral:**Created**  ┃ metric:**step** ┃     metric:**acc** ┃    metric:**loss** ┃ metric:**training_time** ┃ param:**lr**   ┃ param:**momentum** ┃ param:**model_name** ┃ param:**num_classes** ┃ param:**batch_size** ┃ param:**num_epochs** ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━╇━━━━━━╇━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━━━━━━╇━━━━━━━╇━━━━━━━━━━╇━━━━━━━━━━━━╇━━━━━━━━━━━━━╇━━━━━━━━━━━━╇━━━━━━━━━━━━┩
+│ **workspace**               │ -        │   **13** │ **0.86066** │ **0.36708** │        **87.417** │ **0.025** │ **0.5**      │ **squeezenet** │ **2**           │ **8**          │ **2**          │
+│ **main**                    │ **10:27 AM** │    **1** │ **0.88525** │ **0.28284** │        **82.143** │ **0.01**  │ **0.09**     │ **squeezenet** │ **2**           │ **8**          │ **2**          │
+│ │ ╓ a771043 [exp-3d76f] │ 10:46 AM     │    1 │ 0.89754 │ 0.28754 │        82.447 │ 0.025 │ 0.5      │ squeezenet │ 2           │ 8          │ 2          │
+│ │ ╟ 39e3ce6             │ 10:46 AM     │    1 │ 0.89754 │ 0.28754 │        82.447 │ 0.025 │ 0.5      │ squeezenet │ 2           │ 8          │ 2          │
+│ ├─╨ 707a0aa             │ 10:45 AM     │    0 │  0.7459 │ 0.63368 │        28.104 │ 0.025 │ 0.5      │ squeezenet │ 2           │ 8          │ 2          │
+│ │ ╓ c12efe1 [exp-44404] │ 10:43 AM     │    7 │ 0.79918 │  3.2234 │        25.293 │ 0.025 │ 0.5      │ alexnet    │ 2           │ 8          │ 1          │
+│ │ ╟ 302417a (4f6d314)   │ 10:42 AM     │    7 │ 0.79918 │  3.2234 │        25.293 │ 0.025 │ 0.5      │ alexnet    │ 2           │ 8          │ 1          │
+│ │ ╓ 4f6d314 [exp-67bae] │ 10:35 AM     │    6 │ 0.82377 │  2.3404 │        75.994 │ 0.025 │ 0.5      │ alexnet    │ 2           │ 8          │ 2          │
+```
+
+The newest experiment has an accuracy that's significantly different since we
+switched models. That tells us that the hyperparameter values that were good for
+AlexNet might not work the best for SqueezeNet.
+
+So we'll need to run a few experiments to find the best hyperparameter values.
+This time, we'll take advantage of queues in DVC to set up the experiments and
+then run them at the same time. To set up a queue, we'll run this command.
+
+```dvc
+$ dvc exp run --queue -S lr=0.0001 -S momentum=0.9 -S num_epochs=2
+```
+
+Running this sets up an experiment for future execution so we'll go ahead a run
+this command one more time with different values.
+
+```dvc
+$ dvc exp run --queue -S lr=0.001 -S momentum=0.09 -S num_epochs=2
+```
+
+You can check out the details for the queues you have in place by looking at the
+experiments table with `dvc exp show`. You'll see something like this.
+
+```dvctable
```

Comment:
`exp-7aac` in these last two tables looks off to me. It looks like you have squeezenet checkpoints that are running on top of alexnet checkpoints, which doesn't seem like the workflow you have described.

Reply:
Yeah, not sure what I did there but it should be cleared up now!